src/components/Sidebar/CalendarPanel.vue

```diff
@@ -22,19 +24,30 @@
     },
   })
   export default class CalendarPanel extends mixins(SidebarPanelMixin) {
+
+    @Prop({required: false}) readonly notableDates!: any;
+
     private dates: {
       start?: Date;
       end?: Date;
     } = {};
 
+    private defaultDate!: DateItem;
```

Comment:
Is this no longer needed?

Reply:
It's needed to provide a type to the emitter function I think. 
![image](https://user-images.githubusercontent.com/80924593/123290384-3a8c4980-d4df-11eb-9721-fffcebe8f963.png)

Reply:
I wasn't sure if there was another way to provide a type to the function

Reply:
`changePage(shownMonth: DateItem)`

Reply:
Yep that's way better, thanks